settings.py

```diff
@@ -398,25 +399,33 @@
 TERRA_WORKSPACE_CACHE_EXPIRE_SECONDS = os.environ.get('TERRA_WORKSPACE_CACHE_EXPIRE_SECONDS', 300)
 
 SERVICE_ACCOUNT_FOR_ANVIL = None
+SERVICE_ACCOUNT_CREDENTIALS = None
 
 AIRFLOW_API_AUDIENCE = os.environ.get('AIRFLOW_API_AUDIENCE')
 AIRFLOW_WEBSERVER_URL = os.environ.get('AIRFLOW_WEBSERVER_URL')
 
 if TERRA_API_ROOT_URL:
-    SERVICE_ACCOUNT_FOR_ANVIL = subprocess.run(['gcloud auth list --filter=status:ACTIVE --format="value(account)"'],
+    service_account_file = '/.config/service-account-key.json'
+    if not os.path.exists(service_account_file):
+        raise Exception('Error starting seqr - gcloud auth is not properly configured')
```

Comment:
It seems unnecessary to check the file's existence because it will be an exception in the following getting credential operation.